### ISSUE TITLE
Toast is firing warning when used with React 18 alpha

### ISSUE THREAD

**Issue by fc1943s** [chakra-ui/chakra-ui#4360]
### Description

Toast is firing warning when used with React 18 alpha, even when its not being used

### Link to Reproduction

https://codesandbox.io/s/xenodochial-fermi-eekwf

### Chakra UI Version

1.6.5

### Additional Information

The problem is probably happening because of the @reach/alert library, which uses the "old" render function to mount alerts. 

![image](https://user-images.githubusercontent.com/688618/125100717-cb525000-e0af-11eb-9128-188c2d69db3c.png)

![image](https://user-images.githubusercontent.com/688618/125100743-d311f480-e0af-11eb-82c4-44e71df0ae11.png)

![image](https://user-images.githubusercontent.com/688618/125100622-b1187200-e0af-11eb-9282-eb2a4acd7012.png)

![image](https://user-images.githubusercontent.com/688618/125101741-effaf780-e0b0-11eb-9abd-978b803a7dd6.png)

**Comment by devJimmyboy** [comment]
I've been dealing with this warning for a little while now. I remember the last time I looked into Chakra's progress with React 18 Compatibility, I saw contributors messages stating that since it's still in progress, it's not worth it to support Alpha/createRoot when its usage could be changed entirely by the time React 18 is released. If any contributors to Chakra want to reiterate on this I'd appreciate it as well!
